e2e/addons/addons_test.go

```diff
@@ -213,5 +225,21 @@ var _ = Describe("addons flow", func() {
 				Expect(logLine.IngestionTime).NotTo(Equal(0))
 			}
 		})
+
+		It("svc delete should not delete local files", func() {
+			_, err := cli.SvcDelete(svcName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect("./copilot/hello/addons").Should(BeAnExistingFile())
```

Comment:
The tests passes by using `BeAnExistingFile`, will switch this to `BeADirectory`